src/test/ui/associated-type-bounds/const-projection-err.stock.stderr

```diff
@@ -0,0 +1,17 @@
+error[E0271]: type mismatch resolving `<T as TraitWAssocConst>::A == 1`
+  --> $DIR/const-projection-err.rs:14:11
+   |
+LL |     foo::<T>();
+   |           ^ expected `1`, found `<T as TraitWAssocConst>::A`
+   |
+   = note: expected constant `1`
+              found constant `<T as TraitWAssocConst>::A`
```

Comment:
It's strange and sad that we don't normalize this unless `generic_const_exprs` is enabled, but this isn't related to the ICE.